test/livebook_web/live/hub/edit_live_test.exs

```diff
@@ -197,6 +201,45 @@ defmodule LivebookWeb.Hub.EditLiveTest do
     end
   end
 
+  describe "enterprise" do
+    test "updates hub", %{conn: conn} do
+      hub = insert_hub(:enterprise)
+      {:ok, view, _html} = live(conn, Routes.hub_path(conn, :edit, hub.id))
+
+      attrs = %{"hub_color" => "#FF00FF"}
+
+      view
+      |> element("#enterprise-form")
+      |> render_change(%{"enterprise" => attrs})
+
+      refute view
+             |> element("#enterprise-form .invalid-feedback")
+             |> has_element?()
+
+      assert {:ok, view, _html} =
+               view
+               |> element("#enterprise-form")
+               |> render_submit(%{"enterprise" => attrs})
+               |> follow_redirect(conn)
+
+      assert render(view) =~ "Hub updated successfully"
+
+      assert view
+             |> element("#hubs")
+             |> render() =~ ~s/style="color: #FF00FF"/
+
+      assert view
+             |> element("#hubs")
+             |> render() =~ Routes.hub_path(conn, :edit, hub.id)
+
+      assert view
+             |> element("#hubs")
+             |> render() =~ hub.hub_name
```

Comment:
Since we do it a lot, we could simplify:

```suggestion

      hubs_html = view |> element("#hubs") |> render()
      assert hubs_html =~ ~s/style="color: #FF00FF"/
      assert hubs_html =~ Routes.hub_path(conn, :edit, hub.id)
      assert hubs_html =~ hub.hub_name
```